### PR TITLE
Live bar animations

### PR DESCRIPTION
**Pull request by samclaridge**
@jbaxleyiii have a look and see what you think. The bar color and the text animate.

### CODE REVIEW

File: apollos/src/core/components/live/live-css.js [hunk 1]
```diff
@@ -1,10 +1,48 @@
...
     width: "100%",
     zIndex: 1,
   },
+  "live-banner": {
+    ...animation,
+  },
+  "live-text": {
+    ...textAnimation,
+  }
 });
```
Comment (johnthepink) [inline apollos/src/core/components/live/live-css.js]: You should be able to do just:

```js
"live-banner": animation,
"live-text": textAnimation,
```

**Comment by samclaridge** [comment]
@johnthepink fixed!

**Comment by johnthepink** [comment]
Great, I'm happy to merge if @jbaxleyiii likes the designings

**Comment by jbaxleyiii** [comment]
@samclaridge I'll pull it down and check it out tonight!

**Comment by samclaridge** [comment]
@jbaxleyiii I made the color change more subtle by extending the time and making the color slightly darker than primary so it never totally blends in. 

I also added a fade in/out for the text because that's what iOS does for maps and phone when you go to other apps while you're using them. It's kind of in your face, but I think it's appropriate for it being live.

**Pull request merged**